packages/scandipwa/src/component/Router/Router.container.js

```diff
@@ -61,27 +61,31 @@ export const mapStateToProps = (state) => ({
     status_code: state.MetaReducer.status_code
 });
 
+/** @namespace Component/Router/Container/initFunction */
+export const initFunction = async (dispatch) => {
```

Comment:
I see that you are calling this function just once. So why did you extract it?

Reply:
Please give this function more describing name.

Reply:
If possible, please move such function to `utils`, so other components could reuse them.

Reply:
Exported it and created a namespace because of eslint

Reply:
But I see, it was not about the design but that mistake.
So, I'll undo the extraction. :)